src/opentimelineio/linearTimeWarp.h

```diff
@@ -27,6 +28,10 @@ class LinearTimeWarp : public TimeEffect {
         _time_scalar = time_scalar;
     }
 
+    virtual TimeRange output_range(TimeRange input_range,  ErrorStatus* error_status) const {
+        return TimeRange(input_range.start_time(), input_range.duration() / _time_scalar);
```

Comment:
Here you could construct a`TimeTransform` and then use `.applied_to(input_range.duration()` to compute the scaled duration. 

Also, another thing to consider is what happens when `_time_scalar` is zero (like for [FreezeFrame](https://github.com/PixarAnimationStudios/OpenTimelineIO/blob/master/src/opentimelineio/freezeFrame.h)). Perhaps, if it doesn't exist, a unittest for the `FreezeFame` case should be added.